CodeEdit/Documents/WorkspaceDocument.swift

```diff
@@ -20,6 +20,7 @@ import StatusBar
 import TabBar
 
 // swiftlint:disable type_body_length
+// swiftlint:disable file_length
```

Comment:
We will want to break this file up if not now, in the future.

Reply:
I agree with you to break this file up in the future.